src/categorical_algebra/CSets.jl

```diff
@@ -17,6 +18,9 @@ import ..FinSets: FinSet, FinFunction, FinDomFunction, force
 using ...Theories: Category, CatDesc, AttrDesc
 import ...Theories: dom, codom, compose, ⋅, id
 
+import Base.convert
+Base.convert(::Type{Symbol}, str::String) = Symbol(str)
```

Comment:
We shouldn't override the behavior of base types from inside Catlab, since this will affect all loaded packages.

Reply:
Removed in latest commit